test/integration_test.py

```diff
@@ -750,11 +750,24 @@ def _test_repository_files(self, catalog: str):
                     response = self._get_url_json(response['Location'])
 
                 response = self._get_url(response['Location'], stream=True)
-                self._validate_file_response(response, file['format'])
+                self._validate_file_response(response, self._file_format(file))
+
+    def _file_format(self, file: JSON) -> str:
```

Comment:
Let's not call this "format" to avoid confusion with the metadata property of the same name.